docs/03_design/api/openapi.yaml

```
@@ -201,4 +211,37 @@
+                disconnected:
+                  value: { status: disconnected, lastSeenAt: '2025-08-24T02:45:30Z' }
+        '404':
+          description: Bot not found
+          content:
+            application/json:
+              schema: { $ref: '#/components/schemas/ErrorResponse' }
+        '503':
+          description: Bot not connected
+          content:
+            application/json:
+              schema: { $ref: '#/components/schemas/ErrorResponse' }
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)